papers/whitepaper.md

```diff
@@ -0,0 +1,113 @@
+
+These financial incentives combined with a series of optimizations around AI inference will lead to nearly optimal node GPU utilization, allowing the Atoma Network to scale horizontally with the number of nodes, and also to scale vertically with better available hardware per node.
+
+New applications will be built through the Atoma Network that can leverage the immense potential of generative AI models in order to provide a new intelligence layer for Web3 protocols.
+
+The Atoma Network will be a natural framework to support building innovative applications such as protocol based AI generated content, AI market predictions, Web3 AI enhanced wallets specialized for user intent interactions, knowledge bases as public goods, social discussion forums, DAO and Network States governance, etc.
+
+In a world where AI will play the role of our caretaker, tutor, personal assistant, and coworker, it will be crucial that these applications will provide the high trust guarantees around AI inference.
+
+That said, in order for the Atoma Network to become a full intelligence layer powering Web3 protocols, it is required to establish high integrity guarantees around AI inference compute. In practice, this means that AI generated outputs can be proved to have not been tampered with in any possible form; in other words, no malicious actor could have interfered with the correct execution of an AI model.
+
+The sampling consensus protocol, referred to as `Sampling Consensus`, is based on the assumption that nodes are rational actors in a competitive economic environment. The protocol is optimized to achieve very high security guarantees whilst having lower cost compared to other methods (such as zkML or opML).
+
+For the above reasons, the Atoma Network is well positioned to lead the space of decentralized AI compute. We see a future where the Atoma Network will open the doors for a more bright, transparent, innovative, and democratic future where technology is fully aligned with user and community needs.
+
+### Introduction
+
+The past few years have seen the emergence of new generative AI capabilities powered by large language models (LLMs), more powerful hardware models which allow for massive scaling computing power, and a rise of highly scalable blockchains with lower transaction fees that utilize a combination of novel consensus, cryptography, and modularity techniques.
+
+To put this into perspective, McKinsey estimates that generative AI could add the equivalent of between $2.6 trillion to $4.4 trillion annually across the 63 use cases they analyzed, this would increase the impact of all artificial intelligence by 15% to 40%. By comparison, the United Kingdom’s entire GDP in 2021 was $3.1 trillion.
```

Comment:
What does increase the impact mean more specifically? 

I.e the adoption of AI into businesses / our lives ?